addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3509,3 +3509,22 @@ def test_date_reversal_exchange_move(self):
 
                 # Date of the reversal of the exchange move should be the last day of the month/year of the payment depending on the sequence format
                 self.assertEqual(exchange_move_reversal.date, fields.Date.to_date(expected_date))
+
+    def test_invoice_mass_posting(self):
```

Comment:
Maybe add a comment on what you are testing? It is not trivial by reading the test